File: wwwroot/js/models/messMember.js
var MessMember = function (data) {
    var self = this;
    if (data) {
        ko.mapping.fromJS(data, {}, self);
    }
    else {
        self.name = ko.observable();
        self.bnumb = ko.observable();
        self.rank = ko.observable();
        self.appt = ko.observable();
        self.unit = ko.observable();
        self.contactNo = ko.observable();
        self.roomNo = ko.observable();
        self.maritialStatus = ko.observable();
        self.memberStatus = ko.observable();
        self.dateOfEntry = ko.observable();
        self.email = ko.observable();
        self.password = ko.observable();
        self.userId = ko.observable(0);
        self.imagePath = ko.observable();
        self.userRole = ko.observable(1);
    }
}

File: Domain/MessMember.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MessingSystem.Domain
{
    public class MessMember
    {
        [Key]
        public int Id { get; set; }

        public string Name { get; set; }

        public string Bnumb { get; set; }

        public string Rank { get; set; }

        public string Appt { get; set; }

        public string Unit { get; set; }
        public string ContactNo { get; set; }
        public string RoomNo { get; set; }

        public DateTime DateOfEntry { get; set; }

        public string MaritialStatus { get; set; }
        public int MemberStatus { get; set; }
        public int UserId { get; set; }
    }
}
